source/crud/query/distinct.txt

```diff
@@ -162,15 +163,47 @@ with unaccented first letters:
 
    [ "Borgatti's", "Samba Kitchen", "Tanoreen", "Via Carota", "White Bear", "Äpfel" ]
 
-Additional Information
-----------------------
+.. _node-usage-distinct:
 
-For a runnable example of retrieving distinct values, see :ref:`node-usage-distinct`.
+distinct() Usage Example
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-API Documentation
-~~~~~~~~~~~~~~~~~
+.. include:: /includes/crud/example-intro.rst
+
+The following snippet retrieves a list of distinct values for the ``year``
+document field from the ``movies`` collection. It uses a query document to
+match movies that include "Barbara Streisand" as a ``director``.
```

Comment:
S: monospace all values including strings, also avoid using the name of a field as the noun it means. I think that referring to them as fields/field names makes translation and interpretation easier
```suggestion
match movies that include ``"Barbara Streisand"`` in the ``director`` array.
```